apps/expo/src/components/feed-post.tsx

```diff
@@ -246,54 +247,38 @@ const FeedPostInner = ({
           {/* inline "replying to so-and-so" */}
           {displayInlineParent &&
             (item.reply
-              ? (() => {
-                  // use zod to parse the parent post
-                  // since all we want is author + uri
-                  // don't actually care what specific kind of post it is
-                  const parse = z
-                    .object({
-                      author: z.object({
-                        handle: z.string(),
-                        did: z.string(),
-                        displayName: z.string().optional(),
-                      }),
-                      uri: z.string(),
-                    })
-                    .safeParse(item.reply.parent);
-
-                  if (parse.success) {
-                    return (
-                      <Link
-                        href={path(
-                          `/profile/${
-                            parse.data.author.did
-                          }/post/${parse.data.uri.split("/").pop()}`,
-                        )}
-                        asChild
-                        accessibilityHint="Opens parent post"
-                      >
-                        <TouchableWithoutFeedback>
-                          <View className="flex-row items-center">
-                            <MessageCircleIcon size={12} color="#737373" />
-                            <Text
-                              className={cx(
-                                "ml-1 flex-1",
-                                theme.dark
-                                  ? "text-neutral-400"
-                                  : "text-neutral-500",
-                              )}
-                              numberOfLines={1}
-                            >
-                              replying to{" "}
-                              {parse.data.author.displayName ??
-                                `@${parse.data.author.handle}`}
-                            </Text>
-                          </View>
-                        </TouchableWithoutFeedback>
-                      </Link>
-                    );
-                  } else return null;
-                })()
+              ? AppBskyFeedDefs.isPostView(item.reply.parent) && (
+                  <Link
+                    href={path(
+                      `/profile/${
+                        item.reply.parent.author.did
+                      }/post/${item.reply.parent.uri.split("/").pop()}`,
+                    )}
+                    asChild
+                    accessibilityHint="Opens parent post"
```

Comment:
Wow this is turning into quite a huge PR, nice job! 👏

Sorry to be that guy again, but I did notice this `accessibilityHint` that I think should probably be marked for translation?